test/fixtures/label/missingXScale.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
+  tolerance: 0.0075,
+  config: {
+    type: 'bar',
+    options: {
+      scales: {
+        x: {
+          labels: ['January', 'February', 'March', 'April', 'May', 'June', 'July'],
+        },
+        y: {
+          display: false,
+          min: 0,
+          max: 25
+        }
+      },
+      plugins: {
+        annotation: {
+          annotations: {
+            text1: {
+              type: 'label',
+              xScaleID: 'missing',
+              yScaleID: 'y',
+              xValue: 'February',
+              yValue: 2,
+              backgroundColor: 'rgba(33, 101, 171, 0.5)',
+              borderColor: 'rgb(33, 101, 171)',
+              borderWidth: 1,
+              content: ['Missing x scale and', 'then is located to horizontal center'],
+              position: {
+                x: 'center',
+                y: 'center'
+              },
+              xAdjust: 0,
+              yAdjust: -100,
```

Comment:
Why adjust the Y, makes it more confusing too look at since if you look at the image you would expect the text to be centered

Reply:
I added to enable the callout. Anyway I agree with you. I'm removing it.